src/EditorFeatures/Core/IntelliSense/ModelComputation.cs

```diff
@@ -81,21 +81,8 @@ public Task<TModel> ModelTask
         }
     }
 
-    public TModel WaitForController()
-    {
-        ThreadingContext.ThrowIfNotOnUIThread();
-
-        var model = ModelTask.WaitAndGetResult(CancellationToken.None);
```

Comment:
this got rid of another blocking wait, which was the initial reason i was in this space.  but i then discovered just how useless all this code is to begin with...